DESCRIPTION

```diff
@@ -30,7 +30,8 @@ Imports:
     ranger,
     ROCR,
     RSQLite,
-    xgboost
+    xgboost,
+    cluster
```

Comment:
Can we use kmeans functionality from the built-in stats package? Feel pretty strongly here. Is always nice not to introduce more dependencies. This would also help us to keep this functionality simple (as data architects would likely require a super simple tool here). Let me know what which functionality wouldn't be possible...